FirebaseSegmentation/Tests/Unit/SEGInitializationTests.m

```diff
@@ -15,18 +15,27 @@
 #import <XCTest/XCTest.h>
 
 #import "FIRApp.h"
+#import "FIROptions.h"
 #import "FirebaseSegmentation/Sources/Public/FIRSegmentation.h"
 
-@interface SEGInitializationTests : XCTestCase
+@interface FIRSegmentation (ForTest)
+- (instancetype)initWithAppName:(NSString *)appName FIROptions:(FIROptions *)options;
+@end
+
+@interface SEGInitializationTests : XCTestCase {
+  FIRSegmentation *_segmentation;
+}
 
 @end
 
 @implementation SEGInitializationTests
 
 - (void)setUp {
-  // Put setup code here. This method is called before the invocation of each test method in the
-  // class.
-  [FIRApp configure];
+  FIROptions *options = [[FIROptions alloc] init];
```

Comment:
I thought `init` was unavailable and you have to use `initWithGoogleAppID:GCMSenderID:`?